cmd/kube-scheduler/app/options/options_test.go

```diff
@@ -226,6 +227,36 @@ plugins:
     - name: baz
 pluginConfig:
 - name: foo
+`, configKubeconfig)), os.FileMode(0600)); err != nil {
+		t.Fatal(err)
+	}
+	// v1alpha1 plugin config
+	v1alpha1pluginconfigFile := filepath.Join(tmpDir, "v1alpha1_plugin.yaml")
```

Comment:
name it something like `v1alpha1_postfilter`
Because I'm adding a similar test for #87657. And then it should be fine to not have other endpoints (like reserve), but feel free to keep them.

Reply:
It's updated and squashed.